src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -90,16 +87,16 @@ private TexlParser(IReadOnlyList<Token> tokens, Flags flags, Features features =
         /// </summary>
         /// <param name="script">Script to be parsed.</param>
         /// <param name="parserOptions">Options for parsing an expression.</param>
+        /// <param name="features">Power Fx feature flags.</param>
         /// <returns><see cref="ParseUserDefinitionResult"/>.</returns>
-        public static ParseUserDefinitionResult ParseUserDefinitionScript(string script, ParserOptions parserOptions)
+        public static ParseUserDefinitionResult ParseUserDefinitionScript(string script, ParserOptions parserOptions, Features features = null)

```

Comment:
>ParserOptions parserOptions, Features feature [](http://example.com/codeflow?start=89&length=45)

We have a PFxV1 in both ParserOptions and in Features... Not something that needs to be fixed in this PR, but we should have a follow-up issue to clean this up.